devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -67,7 +68,18 @@ def parseAttrMessage(description) {
 		map = getBatteryPercentageResult(Integer.parseInt(descMap.value, 16))
 	} else if (descMap?.clusterInt == zigbee.ONOFF_CLUSTER && descMap.isClusterSpecific) {
 		map = getButtonEvent(descMap)
-	}
+	} else if(descMap?.clusterInt == 0xFC80) {
+		def buttonNumber
+		buttonNumber = Integer.valueOf(descMap?.command[1].toInteger()) + 1
+       
+		log.debug "Number is ${buttonNumber}"
+		def event = createEvent(name: "button", value: "pushed", data: [buttonNumber: buttonNumber], descriptionText: "pushed", isStateChange: true)
```

Comment:
Does this device support `held`? If not you should modify `supportedButtonValues`